extensions/wikia/AdEngine3/src/setup.js

```diff
@@ -179,6 +179,13 @@ function setupAdContext(wikiContext, isOptedIn = false, geoRequiresConsent = tru
 		if (!isGeoEnabled('wgAdDriverPubMaticOutstreamCountries')) {
 			context.remove('bidders.prebid.pubmatic.slots.INCONTENT_PLAYER');
 		}
+
+		const unbiddedSlotName = hasFeaturedVideo ? 'INCONTENT_PLAYER' : 'FEATURED';
```

Comment:
Can you please change the name to `notBiddedSlotName`? I think there is no verb `to unbid` :)

Reply:
Done 👍